app/services/food_service.rb

```diff
@@ -0,0 +1,16 @@
+class FoodService 
+
+  def self.search_foods(food_item)
+    response = conn.get("/fdc/v1/foods/search?query=#{food_item}&limit=10")
+   
+    data= JSON.parse(response.body, symbolize_names: true)
+    
+  end
+
+
+  def self.conn 
+    Faraday.new(url: 'https://api.nal.usda.gov') do |f|
+      f.params['api_key'] = ENV['food_key']
+    end
+  end
+end
```

Comment:
This all looks really great, nice work! This is the endpoint we had expected y'all to use.